orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/CopyNumberSelector.java

```diff
@@ -87,8 +87,8 @@ private static PurpleGainLoss toFullGain(@NotNull GeneCopyNumber geneCopyNumber)
                 .transcript(geneCopyNumber.transName())
                 .isCanonical(geneCopyNumber.isCanonical())
                 .interpretation(CopyNumberInterpretation.FULL_GAIN)
-                .minCopies(Math.round(Math.max(0, geneCopyNumber.minCopyNumber())))
-                .maxCopies(Math.round(Math.max(0, geneCopyNumber.maxCopyNumber())))
+                .minCopies(Math.abs(geneCopyNumber.minCopyNumber()))
+                .maxCopies(Math.abs(geneCopyNumber.maxCopyNumber()))
```

Comment:
`abs` is not the correct approach here. Purple can generate negative copy numbers, and when we convert them to purpleGainLoss we should floor to 0. A -1.2 copy number should become 0 rather than +1.2

Reply:
Thanks for clarifying. Fixed in all places